lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -101,21 +103,25 @@ describe('AwsInvokeLocal', () => {
       });
     });
 
-    it('it should parse mock file if provided', () => {
+    it('should parse data if it is a json string', () => {
```

Comment:
We have the same test above ("Should parse JSON data if it is provided via CLI")